core/block_svg.js

```diff
@@ -932,12 +932,16 @@ Blockly.BlockSvg.prototype.dispose = function(healStack, animate) {
  */
 Blockly.BlockSvg.prototype.toCopyData = function() {
   var xml = Blockly.Xml.blockToDom(this, true);
-  // Copy only the selected block and internal blocks.
-  Blockly.Xml.deleteNext(xml);
-  // Encode start position in XML.
-  var xy = this.getRelativeToSurfaceXY();
-  xml.setAttribute('x', this.RTL ? -xy.x : xy.x);
-  xml.setAttribute('y', xy.y);
+  // If this block is an insertion marker, then the xml is an empty DocumentFragment
```

Comment:
Can an insertion marker be copied? If not, can we just assert that this isn't an insertion marker and keep the clipboardXml_ type of `!Element`.

Reply:
Hmm, how about if we return null here when it is an insertion marker instead? I think it is probably safe to assume people aren't going to copy an insertion marker but this won't throw an error if someone does so it seems a bit safer.

Reply:
Yep, that would just make it nullable.